spec/spec_helper.rb

```diff
@@ -130,7 +130,9 @@
 
   config.before(:each, :draper_with_helpers) do
     c = ApplicationController.new
-    c.request = ActionDispatch::TestRequest.new({})
+    request = ActionDispatch::TestRequest.new({})
+    request.host = "test.host"
+    c.request = request
```

Comment:
```suggestion
    c.request = ActionDispatch::TestRequest.new({host: "test.host"})
```

Doesn't work?

Also consider adding a comment explaining why this is needed.

Reply:
this was my first try and that did not work. It seems that the passed ENV ist parsed differently than the exposed attribute API.

Reply:
the most thorough comment is in the commit-message, a short version has been added to the most confusing line